frontend/packages/console-shared/src/utils/utils.ts

```
@@ -26,3 +26,6 @@ export const createLookup = <A extends K8sResourceKind>(
   }
   return {};
 };
+
+export const prefixedID = (idPrefix: string, id: string) =>
+  idPrefix && id ? `${idPrefix}-${id}` : null;
```

To make the function "safe", what about returning at least one of the trueish param if any?

---

Is it enough to be unique?

---

No it is not (in case some mistake happens or we get a null entity). Please see the use case in `details-page.tsx`

---

ok, fail fast ... Makes sense